vendor/bat-native-ledger/src/bat/ledger/internal/publisher/publisher.cc

```diff
@@ -140,17 +142,18 @@ void Publisher::SaveVisit(
     return;
   }
 
-  auto server_callback =
-      std::bind(&Publisher::OnSaveVisitServerPublisher,
-                this,
-                _1,
-                publisher_key,
-                visit_data,
-                duration,
-                window_id,
-                callback);
+  auto on_server_info = std::bind(&Publisher::OnSaveVisitServerPublisher,
+      this, _1, publisher_key, visit_data, duration, window_id, callback);
 
-  ledger_->GetServerPublisherInfo(publisher_key, server_callback);
+  ledger_->SearchPublisherList(
```

Comment:
On page visits, we want to search the publisher prefix list before attempting to call `GetServerPublisherInfo` (which could trigger a fetch).